torch/csrc/jit/mobile/import.cpp

```diff
@@ -6,8 +6,8 @@
 #include <torch/csrc/jit/mobile/interpreter.h>
 #include <torch/csrc/jit/mobile/observer.h>
 #include <torch/csrc/jit/runtime/instruction.h>
+#include <torch/csrc/jit/serialization/import.h>
```

Comment:
The definition of `readArchiveAndTensors` is in `jit/serialization/import.cpp`. This file is not included in mobile runtime compilation. That might be the reason CI `pytorch_macos_10_13_py3_lite_interpreter_build_test` fails.